### ISSUE TITLE
Handle large download requests better

### ISSUE THREAD

**Issue by JoeGermuska** [censusreporter/censusreporter#93]
We got an email to the Census Reporter inbox from someone trying to download a table (B19001) for all ZCTA in US. Unsurprisingly, CR doesn't handle this well.

A very low-overhead solution would be to pawn people off on Census Fact Finder in certain cases. 

"Census Reporter is unable to support requests for all [sumlev] in [parent sumlev]. Please use American FactFinder."

The FactFinder bookmark URLs are pretty computable, so we can get people most of the way to where they're going: http://factfinder.census.gov/bkmk/table/1.0/en/ACS/13_5YR/B19001/0100000US.86000

I'd love to just give them what they want, but that's a bigger change.

**Comment by ryanpitts** [comment]
What would be the general framework for such a change? Hand off to a celery task, and send an alert email when the download is ready?

**Comment by JoeGermuska** [comment]
yeah, i think if we were going to do it ourselves instead of passing to Census. 

Maybe we could check a cache before queuing. It would probably have more misses than hits, but some things would be repetitive.

I'm not totally sure that the issue is only in the response generation time. We'd want to make sure that Celery tasks didn't grind the server down as much as real-time requests.

**Comment by JoeGermuska** [comment]
maybe has some relationship to #17

**Comment by iandees** [comment]
We should adjust the current limit so that we can query block groups in county and tracts in state. There should be no case where our limit blocks those.

**Comment by JoeGermuska** [comment]
@iandees do you still think the above is correct? I am never sure that the limit really saves us that much, but 🤷‍♂️ 

States with more than 3500 Census tracts:

state | num
----- | -----
 FL     |  4245
 NY     |  4918
 TX     |  5265
 CA     |  8057

And counties with > 3500 block groups:

 fips  | num  
-------|------
 17031 | 3993
 06037 | 6425
